Refresh.GameServer/Endpoints/Game/Handshake/AuthenticationEndpoints.cs

```diff
@@ -278,6 +281,8 @@ public Response RevokeThisToken(RequestContext context, GameDatabaseContext data
         
         // Revoke the token
         database.RevokeToken(token);
+        // Mark them as disconnected from the presence server
+        database.SetUserPresenceAuthToken(user, null);
```

Comment:
Not expected, but I don't think its particularly bad if that happens, since it gets cleared on game login, and gets cleared when the presence server detects a disconnection. The data of "connected to presence server" is only useful if we know the user is in-game in a lobby (that's how list play now works on refresh-web currently)
Clearing it here is just a "good samaritan" thing to do, rather than required for any use/functionality